components/admin_console/system_user_detail/team_list/team_row.jsx

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Why was this cloned?

Reply:
Different functionality in terms of behavior and I'm not 100% sure they will not diverge more as the design is fleshed out more. As they live in different contexts, I don't think it makes sense to try to have it serve both purposes. I don't know, that may change going forward. If there are reuseable components, we could promote the component up to become a shared widget. There are other UI behaviors such as the row hover actions and row content that I'm not sure will be compatible with each other. The CSS, on the other hand, was equivalent in the spec for the teams panel.

Reply:
I'm 4/5 on this. I've went over the differences and they can be mitigated by adding two, three new props. It's would be a shame to duplicate so much code... Think of our poor bundle size! :)

Reply:
I'm okay with that assuming no other major changes from UX/PM (there could still be), howerver, I'd rather address it after the next phase (adding the actions). Perhaps we file another ticket to merge to two after the next phase? I'd consider that more of a refactor. Is all work complete with the team list? I found a potential bug last week: https://mattermost.atlassian.net/browse/MM-17379